src/test/java/com/microsoft/sqlserver/jdbc/databasemetadata/DatabaseMetaDataTest.java

```diff
@@ -1034,6 +1035,51 @@ public void shouldEscapeSchemaName() throws SQLException {
         }
     }
 
+    @Test
+    @Tag(Constants.xAzureSQLDW)
+    @Tag(Constants.xAzureSQLDB)
+    void testGetSchemasWithAndWithoutCatalog() throws Exception {
+        String dbName = "TestDb_GetSchemas_Inline";
```

Comment:
Use `RandomUtil.getIdentifier("GetSchemas");` to ensure a unique database and that parallel tests don't step on each other.
```suggestion
        String dbName = RandomUtil.getIdentifier("GetSchemas");;
```